src/YoutubeAtomPlayer.tsx

```diff
@@ -75,7 +75,34 @@ const createOnStateChangeListener = (
      */
     const player = event.target;
 
+    /**
+     * the ID of the iframe
+     */
+    const atomId = `youtube-video-${videoId}`;
```

Comment:
Can we inline this? It's only used in this one place so the abstraction is only serving the purpose of giving the value a name but I think in this case it's already clear what this is. 

Perhaps if there was a need to make things clearer you could add a comment above the place where we exclude the current video explaining what we are filtering out?